Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,16 @@
-﻿using System;
+﻿using System.Globalization;
+using System.Linq;
+using System.Threading;
 
 namespace BootCamp.Chapter
 {
     public class Program
     {
         public static void Main(string[] args)
         {
-
+            Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfo("de-DE");
```

Comment:
Probably need to add to the readme that "de-DE" is the expected output culture.

Reply:
Good idea.